tests/components/buttons.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Buttons from "../../src/demos/components/buttons";
```

Comment:
Hey, I have configured a shortcut in `package.json` to avoid these relative imports:

```
  "jest": {
    "moduleNameMapper": {
      "@libs/(.*)": "<rootDir>/src/libs/$1"
    }
  }
```

It allows to write:

```
import Foo from "@libs/components/foo";
```

You should probably do the same for the `demos` package, by updating the `package.json` like this:

```
  "jest": {
    "moduleNameMapper": {
      "@demos/(.*)": "<rootDir>/src/libs/$1",
      "@libs/(.*)": "<rootDir>/src/libs/$1"
    }
  }
```

Then you could write:

```
import Buttons from "@demos/components/buttons";
```